Description: make ints unsigned to match boost library
 argc should be unsigned everywhere, negative values have no sense
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Bug-Debian: http://bugs.debian.org/724100
Last-Update: 2013-10-13

---

--- a/src/mongo/util/cmdline_utils/censor_cmdline_test.cpp
+++ b/src/mongo/util/cmdline_utils/censor_cmdline_test.cpp
@@ -85,7 +85,7 @@
             "KEEP",
             "--servicePasswordFake=KEEP"
         };
-        const int argc = boost::size(argv);
+        const unsigned int argc = boost::size(argv);
         testCensoringArgv(argv, argv, argc);
     }
 
@@ -102,7 +102,7 @@
             "--servicePassword",
             "get out of dodge"
         };
-        const int argc = boost::size(argv);
+        const unsigned int argc = boost::size(argv);
 
         const char* const expected[] = {
             "first",
@@ -134,7 +134,7 @@
             "-servicePassword",
             "get out of dodge"
         };
-        const int argc = boost::size(argv);
+        const unsigned int argc = boost::size(argv);
 
         const char* const expected[] = {
             "first",
@@ -166,7 +166,7 @@
             "KEEP",
             "--servicePasswordFake=KEEP"
         };
-        const int argc = boost::size(argv);
+        const unsigned int argc = boost::size(argv);
         testCensoringVector(argv, argv, argc);
     }
 
@@ -183,7 +183,7 @@
             "--servicePassword",
             "get out of dodge"
         };
-        const int argc = boost::size(argv);
+        const unsigned int argc = boost::size(argv);
 
         const char* const expected[] = {
             "first",
@@ -215,7 +215,7 @@
             "-servicePassword",
             "get out of dodge"
         };
-        const int argc = boost::size(argv);
+        const unsigned int argc = boost::size(argv);
 
         const char* const expected[] = {
             "first",
